src/global.css

```diff
@@ -1,3 +1,23 @@
 @tailwind base;
 @tailwind components;
-@tailwind utilities;
\ No newline at end of file
+@tailwind utilities;
+
+input[type='checkbox']::before {
+  content: '';
+  width: 0.65em;
+  height: 0.65em;
+  transform: scale(0);
+  transition: 60ms transform ease-in-out;
+  box-shadow: inset 1em 1em #fff;
+  transform-origin: bottom left;
+  clip-path: polygon(14% 44%, 0 65%, 50% 100%, 100% 16%, 80% 0%, 43% 62%);
+  cursor: pointer;
+}
+
+input[type='checkbox']:checked::before {
+  transform: scale(1);
+}
```

Comment:
This approach seems reasonable to me. 

Svelte also has a `:global` modifier that we could use: https://svelte.dev/docs#component-format-style. The only benefit that stands out to me is that it may be more idiomatic to Svelte. 

Agreed that we should research whether this would typically be done with utility classes. Maybe we can create our own in a way that aligns with how daisyUI does things?